docs/source/algorithms/CalculatePlaczekSelfScattering-v1.rst

```diff
@@ -0,0 +1,88 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm takes an incident spectrum function and it's derivative obtained
+from :ref:`FitIncidentSpectrum <algm-FitIncidentSpectrum>` and calculates the
```

Comment:
I would relax this statement a bit.
The User could provide the incident spectrum function and derivative without having to fit.
Yet, I definitely think the link to `FitIncidentSpectrum` algorithm should stay.

Maybe something like:

> "This algorithm takes an incident spectrum function and it's derivative and calculates the **time-of-flight** Placzek scattering correction from the workspaces material sample and detector info. [1]_ [2]_ [3]_ For obtaining the incident spectrum from a measurement (ie beam  monitors or calibrant sample), the  :ref:`FitIncidentSpectrum <algm-FitIncidentSpectrum>` can provide the necessary inputs."

I highlight the **time-of-flight** since this is **not** for constant wavelength (ie reactor-based scattering)